nautobot/core/apps/__init__.py

```diff
@@ -367,11 +367,6 @@ def convert_biginteger(field, registry=None):  # pylint: disable=redefined-outer
 
         super().ready()
 
-        # Register jobs last after everything else has been done.
-        from nautobot.core.celery import import_jobs
```

Comment:
Would you mind updating the comment in `nautobot.extras.signals` that refers to import_jobs running at server startup?